release/license_info.sh

```diff
@@ -0,0 +1,34 @@
+#!/usr/bin/env bash
+# Copyright (C) 2018 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+# shellcheck source=common.sh
+
+set -e
+
+SCRIPT_DIR=$(dirname "$(realpath "${BASH_SOURCE[0]}")")
+. "${SCRIPT_DIR}/../common.sh"
+. ./config/config.sh
+
+var_load MIX_VERSION
+var_load CLR_LATEST
+dnf_conf="${MIXER_DIR}/.yum-mix.conf"
+
+generate_license_info() {
+    awk '{print $1}' < "${MIXER_DIR}/update/image/${MIX_VERSION}/os-packages" |
+        xargs dnf --quiet --config "${dnf_conf}" --releasever="${CLR_LATEST}" --installroot=dnf-cache repoquery --queryformat "%{LICENSE}" |
```

Comment:
The only thing left open from previous discussion:
What is the purpose of `--installroot=dnf-cache` here? Why is it required?

Reply:
.yum-mix.conf has a line `cachedir=/var/cache/yum/clear` which was owned by root, probably from other mixer runs in the past on my machine, causing dnf to fail. Instead, create or use a cache dir owned by current user in current directory.
```suggestion
        # DNF config uses cache dir inside /var/cache, since we aren't root use --installroot to create/use a local cache in current dir.
        xargs dnf --quiet --config "${dnf_conf}" --releasever="${CLR_LATEST}" --installroot=dnf-cache repoquery --queryformat "%{LICENSE}" |
```

Reply:
So than you want dnf-cache to be inside ${WORK_DIR} for sure instead of current folder, so it gets discarded between builds instead of being an ever growing folder in the builder machine.

Reply:
Agreed.
```suggestion
        xargs dnf --quiet --config "${dnf_conf}" --releasever="${CLR_LATEST}" --installroot="${WORK_DIR}/dnf-cache" repoquery --queryformat "%{LICENSE}" |
```